common/lib/xmodule/xmodule/x_module.py

```diff
@@ -366,6 +366,14 @@ def display_name_with_default_escaped(self):
         return course_metadata_utils.display_name_with_default_escaped(self)
 
     @property
+    def tooltip_content(self):
```

Comment:
I like that name better, as it really is a title; content could be interpreted as implying some sort of html block or something. Thanks.